javascriptv3/README.md

```diff
@@ -65,6 +65,21 @@ undergoing active development. Refer to
 [this GitHub issue](https://github.com/awsdocs/aws-doc-sdk-examples/issues/4127)
 for more information.
 
+### Build the Docker image
+
+1. Install and run docker on your machine.
+2. Navigate to the same directory as this readme.
+3. Run `docker build -t <image_name> .` where `image_name` is a name you provide for the image.
```

Comment:
3. Run `docker build -t <image_name> .` and replace `image_name` with a name for the image.